components/Link/index.js

```diff
@@ -202,21 +200,23 @@ const Link = ({
 };
 
 Link.defaultProps = {
+	value: undefined,
+	url: undefined,
+	className: undefined,
 	type: '',
 	kind: '',
-
 	placeholder: __('Link text ...', '10up-block-components'),
 };
 
 Link.propTypes = {
-	value: PropTypes.string.isRequired,
```

Comment:
The initial value of an attribute is `undefined` if no default value is set. Therefore the `isRequired` caused a console error.